lib/statesman/adapters/active_record_queries.rb

```diff
@@ -30,6 +34,30 @@ def most_recent_transition_join
 
         private
 
+        def in_state_without_initial(*states)
+          states = states.flatten.map(&:to_s)
```

Comment:
```suggestion

```
Since this is a private method we can be sure that the callers are transforming the states into  strings above and remove the `map` here. The same applies to the `not_in_state` method.

Reply:
Thanks, fixed in https://github.com/gocardless/statesman/pull/343/commits/d3d7ae4da4ea112c8a45705c91164cf6ade12ee8!